CMakeLists.txt

```diff
@@ -1,38 +1,58 @@
+cmake_minimum_required(VERSION 3.5)
+
 project( kode )
-set(CMAKE_MODULE_PATH "${kode_SOURCE_DIR}/cmake/modules")
 
-find_package(KDE4 REQUIRED)
```

Comment:
The commit message is doubly misleading :-)
1) this was depending on KDE4 (kdelibs4 in fact), not 5
2) there's no such thing as "KDE5" anyway (there's KDE Frameworks 5, KDE Plasma 5, and apps...)

Reply:
Hi David,
Thanks for the feedback, I have fixed title.